<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <layout class="QGridLayout" name="okClear"/>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="swap_grid"/>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="3">
        <widget class="QRadioButton" name="LetterR">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QRadioButton" name="LetterB">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="LetterW">
         <property name="text">
          <string>W</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="LetterS">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QRadioButton" name="LetterO">
         <property name="text">
          <string>O</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QRadioButton" name="LetterF">
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QRadioButton" name="LetterD">
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QRadioButton" name="LetterC">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="LetterA">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="LetterQ">
         <property name="text">
          <string>Q</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QRadioButton" name="LetterP">
         <property name="text">
          <string>P</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="LetterE">
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QRadioButton" name="LetterL">
         <property name="text">
          <string>L</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QRadioButton" name="LetterI">
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="LetterZ">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QRadioButton" name="LetterH">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QRadioButton" name="LetterJ">
         <property name="text">
          <string>J</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QRadioButton" name="LetterK">
         <property name="text">
          <string>K</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QRadioButton" name="LetterV">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QRadioButton" name="LetterG">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QRadioButton" name="LetterN">
         <property name="text">
          <string>N</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QRadioButton" name="LetterY">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="LetterX">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QRadioButton" name="LetterT">
         <property name="text">
          <string>T</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QRadioButton" name="LetterM">
         <property name="text">
          <string>M</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QRadioButton" name="LetterU">
         <property name="text">
          <string>U</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="6">
        <widget class="QDial" name="dial_3"/>
       </item>
       <item row="2" column="4">
        <widget class="QDial" name="dial_2"/>
       </item>
       <item row="0" column="6" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="rotortext_3">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="rotortext_1">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="rotortext_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDial" name="dial_1"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
